app/models/minion.rb

```diff
@@ -10,6 +10,13 @@ class NotEnoughMinions < StandardError; end
   # Raised when we fail to assign a role on a minion
   class CouldNotAssignRole < StandardError; end
 
+  # Human readable names for highstate attribute values
+  HIGHSTATES = {
+    pending:  nil,
+    complete: true,
+    failed:   false
+  }.freeze
```

Comment:
I mean something like this: https://github.com/SUSE/pharos/pull/17/files#diff-fa4c43e571bbfe7bd1ae914e5794a39fL13. Then the column can be an integer, and you map with this enum, which is cheaper.